qc_simplex_bam/README.md

```diff
@@ -39,3 +43,29 @@ optional arguments:
   --hsmetrics_minimum_base_quality HSMETRICS_MINIMUM_BASE_QUALITY
   --hsmetrics_coverage_cap HSMETRICS_COVERAGE_CAP
 ```
+### v2.0 qc_simplex_bam_v2.0.cwl
```

Comment:
Here I think we can be agnostic here as this QC module that you are writing, in theory, it can work for any UMI based assay